tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -408,17 +411,39 @@ def test__update_collection__OK(self):
         )
         self.assertEqual(200, response.status_code)
 
-    def test__update_collection_partial_data__OK(self):
-        collection_id = self.generate_collection(self.session).id
-        metadata = {"name": "A new name, and only a new name"}
-        response = self.app.patch(
-            f"/curation/v1/collections/{collection_id}",
-            data=json.dumps(metadata),
-            headers=self.make_owner_header(),
-        )
-        self.assertEqual(200, response.status_code)
-        response = self.app.get(f"curation/v1/collections/{collection_id}")
-        self.assertEqual(response.json["name"], metadata["name"])
+    @patch("backend.corpora.common.entities.Collection.get_doi")
+    # Patch Collection.get_doi to ensure the additional collection.update call is made in the update_collection method
+    def test__update_collection_partial_data__OK(self, get_doi: Mock):
```

Comment:
Also add a test where we try to update some links. I have a feeling there is some unexpected behavior around that.